docs/hypervisors.md

```diff
@@ -0,0 +1,48 @@
+# Hypervisors
+
+* [Hypervisors](#hypervisors)
+    * [Types](#types)
+    * [Determine currently configured hypervisor](#determine-currently-configured-hypervisor)
+    * [Choose a Hypervisor](#choose-a-hypervisor)
+
+## Types
+
+Kata Containers supports multiple hypervisors. Since they offer different
+features and options, Kata Containers provides a separate
+[configuration file](/src/runtime/README.md#configuration)
+for each.
+
+> **Note:**
+>
+> The simplest way to switch between hypervisors is to create a symbolic link
+> to the appropriate hypervisor-specific configuration file.
+
+| Hypervisor | Written in | Architectures | Type | Comment | Configuration file |
+|-|-|-|-|-|-|
+[ACRN](https://projectacrn.org/) | C | `x86_64` | Type 1 | "bare metal" | `configuration-acrn.toml` |
```

Comment:
I would merge the "type" and "comment" collumns, i.e. "Type 1 (bare metal)" and "Type 2 (KVM)".

Reply:
Agreed - done.